packages/api/src/index.js

```diff
@@ -0,0 +1,48 @@
+// @ts-check
```

Comment:
Basically you get the tooling for TypeScript without using it

Reply:
Nice. A great start.